packages/components/src/FormatRelativeDateTime/FormatRelativeDateTime.mdx

```diff
@@ -0,0 +1,46 @@
+---
+name: FormatRelativeDateTime
+menu: Components
+route: /components/format-relative-date-time
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { CivilDate } from "@std-proposal/temporal";
```

Comment:
I think using `CivilDateTime` is a better example, as it allows for usage with hours and min.

Reply:
Good point.